portal-web/test/functional/com/liferay/portalweb/macros/KBArticle.macro

```diff
@@ -52,6 +52,10 @@ definition {
 	}
 
 	macro addCP {
+		WaitForElementPresent(
+			key_header = "Folders and Articles",
+			locator1 = "KnowledgeBase#PAGE_TREE_PANEL_HEADER");
+
```

Comment:
Hi @Luizcarlosqueiroz, adding a wait in this macro will cause a bit of a slow down with a lot of our KB tests including Lima's. What do you think of submitting the change with navigating through URL first and seeing if that fixes the failing test? If not, we can investigate further and use wait as a last resort. If we were to add a wait, it might be better to add it to the test case level too in an attempt to make our tests run faster. Let me know what you think. Thanks!

Reply:
I made the changes, but just to be sure they will work at CI, I trigged the suite in this PR.